src/js/themes/base.d.ts

```diff
@@ -874,7 +887,10 @@ export interface ThemeType {
     size?: string;
     font?: {
       weight?: number | string;
-    }
+    };
+  };
+  radioButtonGroup?: {
+    gap?: string;
```

Comment:
I think it should be GapType instead of string. import it from utils.

Reply:
done